### PR TITLE
Web UI: Convert to Helm.

### PR DESCRIPTION
**Pull request by edunham**
There's probably a better way of doing those endpoint values.

### CODE REVIEW

File: infrastructure/terraform/to_review/web-ui.tf [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-  service_name   = "ui"
-  proc_name      = "web"
-  root_url       = "${var.root_url}"
-  docker_image   = "${local.taskcluster_image_monoimage}"
-  secret_name    = "${module.web_ui_secrets.secret_name}"
-  secrets_hash   = "${module.web_ui_secrets.secrets_hash}"
-  secret_keys    = "${module.web_ui_secrets.env_var_keys}"
-  readiness_path = "/"
-  cpu            = "100m"
-  memory         = "400Mi"
```
Comment (sciurus) [inline infrastructure/terraform/to_review/web-ui.tf:26]: We'll likely go back and adjust all of them alter, but let's port cpu and memory over under the assumption that they changed these from their deployment templates defaults for a reason.

File: infrastructure/k8s/services/web-ui.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+project_name: "taskcluster-web-ui"
+secrets:
+  taskcluster_client_id: "static/taskcluster/web-ui"
```
Comment (sciurus) [inline infrastructure/k8s/services/web-ui.yaml]: I don't see a client id this in the terraform.

File: infrastructure/k8s/services/web-ui.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  port: ""
+  graphql_endpoint: ".Values.graphqlEndpoint" # ${var.root_url}/graphql
+  graphql_subscription_endpoint: ".Values.graphqlSubscriptionEndpoint" # ${var.root_url}/subscription
+  login_strategies: ""
```
Comment (sciurus) [inline infrastructure/k8s/services/web-ui.yaml:6]: You're missing application_name.

**Review by sciurus — CHANGES_REQUESTED** [review]

**Comment by sciurus** [comment]
So with our current system there is no better way of doing the endpoint values. If we need to do this in enough places we could change helmit so we can pass through a template expression instead of magically detecting .Values and wrapping in one.

**Comment by edunham** [comment]
This was the only place we used the endpoint vals.

**Review by sciurus — APPROVED** [review]

**Pull request merged**